pydocstringformatter/formatting/base.py

```diff
@@ -5,6 +5,19 @@
 class Formatter:
     """Base class for docstring formatter"""
 
+    optional: bool = False
+
+    def name(self, delimiter: str = "-") -> str:
+        """Return the class name as a list of word."""
+        result = ""
+        class_name = self.__class__.__name__.replace("Formatter", "")
+        for i, char in enumerate(class_name):
+            if char.upper() == char and i:
+                result += f"{delimiter}{char.lower()}"
+            else:
+                result += char.lower()
+        return result
```

Comment:
Isn't this a bit overcomplicated? How about just adding a `name` attribute to each class, like `optional`. That way we have some more freedom in option names and class names.
It would also allow double names like "--fix-capitalization" and "--f-c" (not that I want to add --f-c, just an example)

Reply:
I see your point, it feel complicated now but it also means we have nothing to do later and class name is consistent with option name. I'll let you choose and I'll do the modification :)

Reply:
Let's do an attribute, it gives us a little more configurability while only requiring one additional lines per class.